src/Uno.UI.Wasm/ts/Windows/Devices/Midi/MidiOutPort.ts

```diff
@@ -0,0 +1,21 @@
+﻿namespace Windows.Devices.Midi {
+	export class MidiOutPort {
+		public static sendDefault(encodedDeviceId: string) {
+			var midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
+			var deviceId = decodeURIComponent(encodedDeviceId);
+			var noteOnMessage = [0x90, 60, 0x7f];    // note on, middle C, full velocity
+			var output = midi.outputs.get(deviceId);
+			output.send(noteOnMessage);  //omitting the timestamp means send immediately.
+			output.send([0x80, 60, 0x40], window.performance.now() + 1000.0); // Inlined array creation- note off, middle C,  
+			// release velocity = 64, timestamp = now + 1000ms.
+		}
+
+		public static sendNoteMessage(encodedDeviceId: string, messageType: number, noteNumber: number, velocity: number) {
+			var midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'midi' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)